.github/ISSUE_TEMPLATE.md

```diff
@@ -1,58 +1,20 @@
-#### Do NOT include any confidential information, this issue will be created on a public repo
-#### This section is completed by the issue reporter
-##### On which version(s) do you encounter this issue?
-###### Please provide the output to 'git describe --all' for  all deployments where you have confirmed it occurs.
-
-##### What type of issue is this? Please tick the appropriate box.
```

Comment:
Would this be worth leaving in if we want folks to be able to log a feature request? I'm keen on moving feature requests to a specs repository elsewhere (or something else) which could have it's own ``ISSUE_TEMPLATE.md``.